src/main/java/org/dependencytrack/resources/v1/ProjectResource.java

```diff
@@ -555,7 +555,6 @@ public Response createProject(final Project jsonProject) {
                     LOGGER.error("Failed to create project %s".formatted(jsonProject), e);
                     throw new ServerErrorException(Response.Status.INTERNAL_SERVER_ERROR);
                 }
-                qm.updateNewProjectACL(project, principal);
```

Comment:
This call caused a unique constraint violation because access teams are already assigned earlier in the method. Essentially this call has been creating duplicate records before.